tests/test_agents.py

```diff
@@ -16,12 +15,20 @@
 import elfpy.time as time
 import elfpy.types as types
 from elfpy.agents.get_wallet_state import get_wallet_state
+from elfpy.agents.policies.init_lp import InitializeLiquidityAgent
+from elfpy.agents.policies.lp_and_withdraw import LpAndWithdrawAgent
+from elfpy.agents.policies.no_action import NoActionAgent
+from elfpy.agents.policies.random_agent import RandomAgent
+from elfpy.agents.policies.single_long import SingleLongAgent
+from elfpy.agents.policies.single_lp import SingleLpAgent
+from elfpy.agents.policies.single_short import SingleShortAgent
```

Comment:
maybe we can put these in a collection of agents, that would be similar to importing their Policy class previously

Reply:
Yeah something like that. I think we could define an agent garden in the init that aggregates them and makes them addressable by string